private/gen/data/datawkt/datawkt.gen.go

```diff
@@ -64,6 +64,81 @@ var (
 		},
 	}
+
+	messageNameToFilePath = map[string]string{
```

Comment:
QQ: would this have been easier to just emit these to a separate generated file from the blob above? I naively would have thought we'd leave the existing generator alone for the data blob and just add another generated file...